git_setup_windows/git_setup_windows.md

```diff
@@ -0,0 +1,339 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  0.0.1
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Setting Up Git on Windows
+
+comment: This module provides recommendations and examples to help new users configure git on their Windows computer for the first time.
+
+long_description: If you're ready to start using the Git version control system, this lesson will walk you through how to get set up. This lesson should be a good fit for people who already have an idea of what version control is but may not have any experience using it yet. No previous experience with Git is expected. This lesson is specific to Windows machines, If you are using Mac or Linux, please follow along with the [set-up guide for those computers]( ).
+
+estimated_time: 25 min
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Download `git` onto a Windows computer
```

Comment:
I removed "download" as learning objective, particularly since downloading it has now been moved into the "lesson prep" section. It is a significant amount of lesson prep, but it isn't actually stuff we want them to _learn_ as much as stuff they need to do in order to learn about config and global.